actor/builtin/paymentbroker/paymentbroker_test.go

```diff
@@ -825,6 +825,60 @@ func TestPaymentBrokerCancel(t *testing.T) {
 	assert.Equal(t, types.NewBlockHeight(10100), channel.Eol)
 }
 
+func TestPaymentBrokerCancelFailsAfterSuccessfulRedeem(t *testing.T) {
+	tf.UnitTest(t)
+
+	addrGetter := address.NewForTestGetter()
+	toAddress := addrGetter()
+	method := "paramsNotZero"
+	addrParam := addrGetter()
+	sectorIdParam := uint64(6)
+	payerParams := []interface{}{addrParam, sectorIdParam}
+	blockHeightParam := types.NewBlockHeight(43)
+	redeemerParams := []interface{}{blockHeightParam}
+
+	sys := setup(t)
+	require.NoError(t, sys.st.SetActor(context.TODO(), toAddress, actor.NewActor(pbTestActorCid, types.NewZeroAttoFIL())))
```

Comment:
baby nit: call this `context.Background()` unless we intend to alter this context later. Code should use context.TODO when it's unclear which Context to use or it is not yet available. Ditto this below.